### PR TITLE
murz's IsAwesome Goodie

### PR DESCRIPTION
**Pull request by murz**
image

**What does your Instant Answer do?**
It's my noobie "IsAwesome" goodie.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It solves the problem of wanting to know if I completed the DuckDuckHack goodie tutorial.

**What are some example queries that trigger this Instant Answer?**
duckduckhack murz

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Pretty much no one

```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/murz.pm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package DDG::Goodie::IsAwesome::murz;
+# ABSTRACT: Write an abstract here
+# Start at https://duck.co/duckduckhack/goodie_overview if you are new
+# to instant answer development
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/murz.pm]: These comments can be removed.

**Comment by murz** [comment]
@MrChrisW  removed, thanks!

File: lib/DDG/Goodie/IsAwesome/murz.pm [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package DDG::Goodie::IsAwesome::murz;
+
+use DDG::Goodie;
+
+zci answer_type => "is_awesome_murz";
+zci is_cached   => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/murz.pm]: This comment can also be removed :smile:

File: lib/DDG/Goodie/IsAwesome/murz.pm [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+name "IsAwesome murz";
+description "My first Goodie, it lets the world know that murz is awesome";
+primary_example_queries "duckduckhack murz";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/murz.pm";
+attribution github => ["GitHubAccount", "murz"];
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/murz.pm]: Looks like this needs updating, the format is usually. 
`attribution github => ["murz", "First Name or Full Name"];`

**Comment by MrChrisW** [comment]
@murz Thanks a lot for contributing, we really appreciate it :smiley:. Congrats on making your first Goodie! :tada: 

Now that you've made your first Goodie, we really hope you'll continue to contribute and make more Instant Answers! Please checkout our [Ideas forum](https://duck.co/ideas) if you're unsure about what to build. There's also lots of GitHub Issues in our repos that needs someone to tackle them!

**Comment by moollaza** [comment]
@murz congrats! It's live: https://duckduckgo.com/?q=duckduckhack+murz&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**